src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -324,6 +326,10 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->validate()
+                            ->ifTrue(function ($v) { return isset($v['supports']) && isset($v['support_strategy']); })
+                            ->thenInvalid('"supports" and "support_strategy" cannot be used together.')
```

Comment:
Have not seen other examples for that yet, but of course I can add a second validation for both being !isset()?